src/clp_ffi_js/ir/UnstructuredIrStreamReader.cpp

```diff
@@ -80,7 +80,13 @@ auto UnstructuredIrStreamReader::get_filtered_log_event_map() const -> FilteredL
     return FilteredLogEventMapTsType{emscripten::val::array(m_filtered_log_event_map.value())};
 }
 
-void UnstructuredIrStreamReader::filter_log_events(LogLevelFilterTsType const& log_level_filter) {
+void UnstructuredIrStreamReader::filter_log_events(
+        LogLevelFilterTsType const& log_level_filter,
+        [[maybe_unused]] std::string const& kql_filter
+) {
+    SPDLOG_WARN(
+            "KQL filters aren't supported for unstructured IR streams, so they're being ignored."
+    );
```

Comment:
ah i understand why the doc was written that way now. can we check the value of this before logging the warning to see if users provide a non-null value or empty string (depending on how emscripten maps such values to std::string).